### PR TITLE
Calendar: describe weekNumberRule option

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md [hunk 1]
```diff
@@ -6,7 +6,196 @@ default: 'auto'
...
-<!-- Description goes here -->
\ No newline at end of file
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
+}
+
+This property can take on one of the following values:
+
+- `firstDay`    
+The first week of the year is the week that contains January 1.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md]: It does not important which year is this. It isn't necessarily the "current" year.

```suggestion
The first week of a year is the week that contains January 1. 
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md [hunk 2]
```diff
@@ -6,7 +6,196 @@ default: 'auto'
...
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
+}
+
+This property can take on one of the following values:
+
+- `firstDay`    
+The first week of the year is the week that contains January 1. 
+
+- `firstFourDays`    
+The first week of the year is the week that starts on one of the first four week days: Monday, Tuesday, Wednesday, or Thursday. This rule is specified by the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard. If the first week of the year begins on a Friday, Saturday, or Sunday, this week is counted as the last week of the previous year.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md]:
```suggestion
The first week of a year is the week that starts on one of the first four week days: Monday, Tuesday, Wednesday, or Thursday. This rule is defined by the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard. If the first week of the year begins on a Friday, Saturday, or Sunday, this week considered the last week of the previous year.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md [hunk 3]
```diff
@@ -6,7 +6,196 @@ default: 'auto'
...
+ 
+- `fullWeek`
+The first week of the year is the week that begins with a day that matches the [firstDayOfWeek](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#firstDayOfWeek) option value.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md]:
```suggestion
The first week of a year is the week that begins with a day that matches the [firstDayOfWeek](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#firstDayOfWeek) option value.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md [hunk 4]
```diff
@@ -6,7 +6,196 @@ default: 'auto'
...
+ 
+- `auto` (default)    
+The week calculation rule depends on the locale. If a week starts on Monday, the `firstFourDays` rule is applied. Otherwise, the `firstDay` rule is in effect.
+
+If you want to implement your own calculation rule, use the [cellTemplate](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#cellTemplate) function:
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md]:
```suggestion
If you want to implement your own week number calculation rule, use the [cellTemplate](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#cellTemplate) function:
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

File: api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md [hunk 5]
```diff
@@ -6,7 +6,196 @@ default: 'auto'
...
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
+}
+
+This property can take on one of the following values:
+
+- `firstDay`    
+The first week of a year is the week that contains January 1. 
+
+- `firstFourDays`    
+The first week of a year is the week that starts on one of the first four week days: Monday, Tuesday, Wednesday, or Thursday. This rule is defined by the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard. If the first week of the year begins on a Friday, Saturday, or Sunday, this week considered the last week of the previous year.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md]:
```suggestion
The first week of a year is the week that starts on one of the first four weekdays: Monday, Tuesday, Wednesday, or Thursday. This rule is defined by the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard. If the first week of the year begins on a Friday, Saturday, or Sunday, this week is considered the last week of the previous year.
```

**Pull request merged**